examples/go-xdp-counter/main.go

```diff
@@ -82,7 +82,7 @@ func main() {
 			c := gobpfd.NewLoaderClient(conn)
 
 			loadRequest := &gobpfd.LoadRequest{
-				Location:    paramData.BytecodeLocation,
+				Location:    &gobpfd.LoadRequest_File{File: paramData.BytecodeLocation},
```

Comment:
I think is assuming a file, but the syntax before took a file or image:
```
./go-xdp-counter -iface vethdd340d3 -location image://quay.io/bpfd-bytecode/go-xdp-counter:latest
```
OR
```
./go-xdp-counter -iface vethdd340d3 -location file:///home/bmcfall/src/bpfd/examples/go-xdp-counter/bpf_bpfel.o
```

Reply:
Ok I opened an issue #413 but ended up taking a stab at fixing this here @Billy99